frontend/src/component/PublicProfile/ProfileMainBar.styl

```diff
@@ -0,0 +1,19 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+.profile__mainBar__info
+  &__wrapper
+    width 90%
```

Comment:
Why not 100%?
(same question in `@media (max-width: max-xs)`)

If it is because 100% is too large, is there a way to avoid this overflow by using a (fixed) margin?

Reply:
I fixed this in the last commit